### ISSUE TITLE
Clicking on an error line in the build log while multi cursor is enabled doesn't turn off multi cursor mode

### ISSUE THREAD

**Issue by ggnkua** [slynch8/10x#1437]
The result is that one of the cursors will jump to the error, while the rest will just hang around

**Comment by stan680** [comment]
Yeah, I think this is likely the same issue as #931 
`GotoNextBuildError` / `GotoPrevBuildError` have this problem too, but with those you can work around it like this fwiw:
```
# Go to the previous build error
Shift Alt N:            N10X.Editor.ClearSelection() or N10X.Editor.ExecuteCommand('GotoPrevBuildError')

# Go to the next build error
Alt N:                  N10X.Editor.ClearSelection() or N10X.Editor.ExecuteCommand('GotoNextBuildError')
```

**Comment by slynch8** [comment]
fixed (0.35.95)
